lametro/models.py

```diff
@@ -687,3 +684,22 @@ def related_files(self):
                 related.extend(bill_packet.related_files)
 
         return related
+
+
+class LAMetroSubject(models.Model):
+    class Meta:
+        unique_together = ['guid', 'name']
+
+    name = models.CharField(max_length=256, unique=True)
+    guid = models.CharField(max_length=256, blank=True, null=True)
+
+    def __str__(self):
+        if self.guid is not None:
+            return '{0} ({1})'.format(self.name, self.guid)
+
+        else:
+            return self.name
+
+    @property
```

Comment:
isn't a blocker, but would we want to have the relation between the subject and bills be a foreign key instead of a dynamic property?

Reply:
Yes, a `ManyToMany` field would be more appropriate. That said, this functions as a lookup table between subject and GUID. We don't have any need to get at bills this way right now, so I removed the property. We can add and populate the relationship in the future, if such a need arises.